### DISCUSSION TITLE
Aula Padronização - Fundamentos do ASP.NET 6

### DISCUSSION THREAD

**Discussion by teixeiralex** [balta-io/2811#35] [Q&A] [answered]
Por gentileza se puder me tirar uma dúvida, na aula de padronização você menciona que na propriedade Errors é possível criar um tipo complexo pra substituir o <string> da List que poderia até ser chamado mesmo de ErrorViewModel. Como seria essa implementação, se puder me dar um pouco mais de detalhe por favor.

Obrigado desde já!

**Comment by andrebaltieri** [comment, marked as answer]
@teixeiralex tem uma implementação que uso chamada Flunt:
https://github.com/andrebaltieri/Flunt

Ela está descrita no curso Modelando Domínios Ricos e Refatorando para testes de unidade.
